Content/SandboxHierarchy.md

```diff
@@ -2,13 +2,13 @@
 
 ### Overview
 
-When sandboxed programs create (or modify) objects, such as files, some object must in fact be created. Sandboxie creates these objects out of the way, to protect the system from harmful changes. But these objects must reside somewhere in the system. This page describes where various types of sandboxed objects are placed.
+When sandboxed programs create (or modify) objects, such as files, in fact, some kind of project should be created. Sandboxie creates these objects out of the way, to protect the system from harmful changes. But these objects must reside somewhere in the system. This page describes where various types of sandboxed objects are placed.
 
 Beginning with version 2.80 of Sandboxie, the layout of the sandbox is not tied to computer-specific device names and account names. See [Portable Sandbox](PortableSandbox.md) for more information.
 
 ### Files
 
-Files are created in the sandbox folder. The hierarchy is as follows:
+Files are created in the sandbox folder in the following hierarchy:

```

Comment:
```suggestion
Files are created in the Sandbox folder according to the following hierarchy:
```

Reply:
2) I would also suggest to replace the remaining `sandbox folder` references in the page with `Sandbox folder`